Automattic/wp-calypso
# SPT: fix containment issue causing permanent double scrollbar

#### Changes proposed in this Pull Request

This PR aims to fix an [issue](https://github.com/Automattic/wp-calypso/issues/39799) where the SPT preview would always show double scrollbars even when the content was shorter than the preview viewport.

The issue seems to be caused by what appears to be a rendering bug. The margins of the top and bottom blocks aren't contained in the parent element. A known CSS remedy for these type of rendering bugs is adding overflow:hidden to the parent container leading to the margins being contained as expected.

Fixes https://github.com/Automattic/wp-calypso/issues/39799

#### Screenshots

**Before**

<img width="1325" alt="Screenshot 2020-03-02 at 12 55 17" src="https://user-images.githubusercontent.com/1562646/75676052-c219cd80-5c88-11ea-9962-647df1db2680.png">


**After**

<img width="1324" alt="Screenshot 2020-03-02 at 12 55 32" src="https://user-images.githubusercontent.com/1562646/75676068-c8a84500-5c88-11ea-953d-2de9b708ab4b.png">


#### Testing instructions

The key things to check are:

* Is the double scrollbar gone after applying this fix when the content is shorter than the preview area.
* Is it still possible to scroll when the content is longer than the preview area.

This requires the FSE Plugin to be active on your WP install. You can find guidance for this at `PaYJgx-sW-p2`. You can test on local WP instance or WPCom (assuming you have sandbox syncing setup).

* Activate FSE Plugin.
* Create new Page.
* See SPT selector.
* Click any template to show the preview
